evalml/model_understanding/graphs.py

```diff
@@ -890,3 +890,25 @@ def graph_prediction_vs_actual_over_time(pipeline, X, y, dates):
                         yaxis={'title': 'Target Values and Predictions'})
 
     return _go.Figure(data=data, layout=layout)
+
+
+def get_linear_coefficients(estimator, features=None):
```

Comment:
Does this show up in the API reference? If not we should add it in.

Reply:
Added it!